packages/kit/src/views/AccountManagerStacks/pages/AccountSelectorStack/WalletDetails/WalletDetailsHeader/index.tsx

```
@@ -63,62 +61,51 @@ export function WalletDetailsHeader({
-          testID="account-edit-button"
-          variant="tertiary"
-          alignSelf="flex-start"
-          $gtMd={{ top: '$0.5' }}
-          onPress={onEditButtonPress}
-          {...(editMode && {
-            color: '$textInteractive',
-            icon: 'CheckLargeOutline',
-            iconColor: '$iconSuccess',
+      renderAvatar={
+        <Stack
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)